packages/frontend/editor-ui/src/features/logs/composables/useChatState.ts

```diff
@@ -232,6 +232,17 @@ export function useChatState(isReadOnly: boolean): ChatState {
 		window.open(route.href, '_blank');
 	}
 
+	watch(
+		() => workflowsStore.workflowId,
+		() => {
+			if (workflowsStore.workflowId !== PLACEHOLDER_EMPTY_WORKFLOW_ID) {
+				return;
+			}
+
+			refreshSession();
```

Comment:
Should it not always refresh the session when `workflowId` changes, not only when it's an empty workflow?